libs/hdf-converters/src/jfrog-xray-mapper.ts

```diff
@@ -161,16 +163,16 @@ export class JfrogXrayMapper extends BaseConverter {
       }
     ],
     passthrough: {
-      raw: {
-        transformer: (
-          data: Record<string, unknown>
-        ): Record<string, unknown> => {
-          return data;
-        }
+      transformer: (data: Record<string, any>): Record<string, unknown> => {
```

Comment:
don't use any since it disables the type system

Reply:
ditto on the comment about any's

Reply:
please see my response there